File: 30 Days of Code/Day 5 Loops/Solution.java
import java.util.Scanner;

public class Solution {
    public static void main( String[ ] args ) {
        Scanner scanner = new Scanner( System.in );

        int n = scanner.nextInt( );

        for( int idx = 1; idx <= 10; idx++ ) {
            System.out.println( n + " x " + idx + " = " + n * idx );
        }

        scanner.close( );
    }
}

File: 30 Days of Code/Day 5 Loops/Solution.cs
using System;

public class Solution {

    public static void Main( String[ ] args ) {
        int n = Convert.ToInt32( Console.ReadLine( ) );

        for( int idx = 1; idx <= 10; idx++ ) {
            Console.WriteLine( $"{n} x {idx} = {n * idx}" );
        }
    }
}